addons/website_forum/views/website_forum.xml

```diff
@@ -142,129 +127,205 @@
-                    <span t-field="forum.name"/>
-                </a>
-                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#oe-help-navbar-collapse">
-                    <span class="navbar-toggler-icon"/>
-                </button>
-                <div class="collapse navbar-collapse" id="oe-help-navbar-collapse">
-                    <ul class="navbar-nav">
-                        <li class="nav-item">
-                            <a t-attf-href="/forum/#{ slug(forum) }?{{ keep_query( 'search', 'filters', sorting='relevancy desc') }}"
-                                t-attf-class="nav-link #{sorting == 'relevancy desc' and 'active' or ''}">Trending</a>
-                        </li>
-                        <li class="nav-item">
-                            <a t-attf-href="/forum/#{ slug(forum) }?{{ keep_query( 'search', 'filters', sorting='create_date desc') }}"
-                                t-attf-class="nav-link #{sorting == 'create_date desc' and 'active' or ''}">Newest</a>
-                        </li>
-                        <li class="nav-item">
-                            <a t-attf-href="/profile/users"
-                                t-attf-class="nav-link #{searches.get('users') and 'active' or ''}">People</a>
-                        </li>
-                        <li class="nav-item">
-                            <a t-attf-href="/forum/#{ slug(forum) }/tag"
-                                t-attf-class="nav-link #{searches.get('tags') and 'active' or ''}">Tags</a>
-                        </li>
-                        <li class="nav-item">
-                            <a t-attf-href="/profile/ranks_badges?badge_category=forum"
-                                t-attf-class="nav-link #{searches.get('badges') and 'active' or ''}">Badges</a>
-                        </li>
-                        <li class="nav-item" t-if="user.karma >= forum.karma_moderate">
-                            <a t-attf-href="/forum/#{ slug(forum) }/flagged_queue"
-                                t-attf-class="nav-link #{flagged_queue_active and 'active' or ''}">Spam</a>
-                        </li>
-                    </ul>
-                    <ul class="navbar-nav ml-auto">
-                        <li t-if="not is_public_user" class="nav-item">
-                            <a t-attf-href="/profile/user/#{uid}?forum_id=#{forum.id}" t-attf-class="text-dark font-weight-bold nav-link#{searches.get('my_profile') and ' active' or ''}">
-                                <i class="fa fa-diamond text-secondary" /> <t t-esc="user_id.karma"/>
+    <div class="navbar navbar-expand-sm navbar-light">
+        <div class="container flex-wrap flex-md-nowrap">
+            <a t-if="forums_count and forums_count > 1" class="btn btn-light border mr-2" href="/forum/" title="All forums">
+                <i class="fa fa-chevron-left mr-1"/>All Forums
```

Comment:
I would keep it on a specific question also.
What do you think?

Reply:
The ">All Forums" button? Yep, actually it's supposed to be visible in all pages but I'm not sure how to do that. I'll try again later

Reply:
Yes the All Forums btn!

Reply:
I'm not able to do this, I'll add a note for the devs